src/DssSpell.t.sol

```diff
@@ -911,28 +911,153 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     // SPELL-SPECIFIC TESTS GO BELOW
+    bytes32           constant  SRC_ILK       = "PSM-USDC-A";
+    bytes32           constant  DST_ILK       = "LITE-PSM-USDC-A";
+    PsmAbstract       immutable srcPsm        = PsmAbstract(      addr.addr("MCD_PSM_USDC_A"));
+    LitePsmLike       immutable dstPsm        = LitePsmLike(      addr.addr("MCD_LITE_PSM_USDC_A"));
+    address           immutable pocket        =                   addr.addr("MCD_LITE_PSM_USDC_A_POCKET");
+    GemAbstract       immutable gem           = GemAbstract(      addr.addr("USDC"));
+    uint256           constant  srcKeep       = 0;
+    uint256           constant  dstWant       = type(uint256).max;
+    uint256           constant  dstBuf        = 400 * MILLION * WAD;
+    uint256           constant  dstMaxLine    = 10 * BILLION * RAD;
+    uint256           constant  dstGap        = 400 * MILLION * RAD;
+    uint256           constant  dstTtl        = 12 hours;
+
+    uint256           constant  srcTin        = 0;
+    uint256           constant  srcTout       = 0;
+
+    // this is declared here because it was the only way to avoid "stack too deep" errors
+    uint256 pDaiSum;
+
+    function test_LITE_PSM_USDC_A_MigrationPhase3() public {
+        (uint256 psrcInk, uint256 psrcArt) = vat.urns(SRC_ILK, address(srcPsm));
+        uint256 psrcVatGem = vat.gem(SRC_ILK, address(srcPsm));
+        uint256 psrcGemBalance = gem.balanceOf(address(srcPsm.gemJoin()));
+        (uint256 pdstInk, uint256 pdstArt) = vat.urns(DST_ILK, address(dstPsm));
+        uint256 pdstVatGem = vat.gem(DST_ILK, address(dstPsm));
+        uint256 pdstGemBalance = gem.balanceOf(address(pocket));
+        uint256 pvice = vat.vice();
+        uint256 ppauseSin = vat.sin(pauseProxy);
+
+        uint256 expectedMoveWad = _min(dstWant, _subcap(psrcInk, srcKeep));
+
+        // This is a hack:
+        pDaiSum = vat.dai(address(pot)) + vat.dai(address(susds));
```

Comment:
I think this needs to check dai amout of `usdsJoin` as `join.exit` will be called as soon as `vat.suck` is called. 

Please add note comment above to explain this :)

Reply:
this has been fixed

Reply:
Sorry for not clarifying enough here. After adding this the test was still failing and I'm not sure using `susds` is correct here. 
In `susds.drip`, after calling [`vat.suck`](https://etherscan.io/address/0x4e7991e5c547ce825bdeb665ee14a3274f9f61e0#code#SUsds.sol#L221) which updates `vat.vice`, [`usdsJoin.exit`](https://etherscan.io/address/0x4e7991e5c547ce825bdeb665ee14a3274f9f61e0#code#F1#L222) is called. And I think this will make `dai.balanceOf(usds)` to 0. So we may need to check `dai.balanceOf(usdsJoin)` instead.